pkg/daemon/ceph/osd/agent.go

```diff
@@ -58,21 +59,24 @@ type OsdAgent struct {
 	kv                *k8sutil.ConfigMapKVStore
 	configCounter     int32
 	osdsCompleted     chan struct{}
+	prepareOnly       bool
 }
 
 func NewAgent(context *clusterd.Context, devices string, usingDeviceFilter bool, metadataDevice, directories string, forceFormat bool,
-	location string, storeConfig config.StoreConfig, cluster *mon.ClusterInfo, nodeName string, kv *k8sutil.ConfigMapKVStore) *OsdAgent {
+	location string, storeConfig config.StoreConfig, cluster *mon.ClusterInfo, nodeName string, kv *k8sutil.ConfigMapKVStore, prepareOnly bool) *OsdAgent {
 
 	return &OsdAgent{devices: devices, usingDeviceFilter: usingDeviceFilter, metadataDevice: metadataDevice,
 		directories: directories, forceFormat: forceFormat, location: location, storeConfig: storeConfig,
 		cluster: cluster, nodeName: nodeName, kv: kv,
 		procMan: proc.New(context.Executor), osdProc: make(map[int]*proc.MonitoredProc),
+		prepareOnly: prepareOnly,
```

Comment:
The indention seems off here, could you check if `go fmt`/`gofmt` changes something here (in my quick testing the indention for all "keys" here was moved to one layer).

Reply:
no change, which go version you are using?

```console
# go version
go version go1.10.1 linux/amd64
# gofmt -w pkg/daemon/ceph/osd/agent.go 
# git diff
# 
```

Reply:
Oh now I see why I thought they should be on one layer.
There are multiple variable on one line, could you change it so that on every line there is one variable?
At least I prefer having one variable per line and also a few lines further down https://github.com/rook/rook/pull/1698/files#diff-d422456f63710f6140fbb62ba3b6ae8aR475 also only has one variable per line (and I can think of other locations where it's one variable per line).